### PR TITLE
Map dependencies' names (version 1)

### PR DESCRIPTION
**Pull request by tzoiker**
It would be nice to be able to inject the same dependency with different names to different services. For example
- You have parent service `ParentService` with parameter `bar`
- Inherit with `ChildService`
- Inject some existing dependency `foo` as `bar`

### CODE REVIEW

File: aiomisc_dependency/__init__.py [hunk 1]
```diff
@@ -13,9 +17,38 @@ def dependency(func):
...
+
+async def inject(target, *, dependencies_list, dependencies_map):
+    dependencies_list = dependencies_list or []
+    dependencies_map = dependencies_map or {}
+    if (
+        isinstance(dependencies_list, Mapping) or
+        not isinstance(dependencies_map, Mapping)
+    ):
+        raise ValueError(
+            '__dependencies__ must be not be mapping, '
```
Comment (dizballanze) [inline aiomisc_dependency/__init__.py]: Typo: `must not be mapping`.
Better message: `__dependencies__ must be Sequence, not Mapping`

**Review by dizballanze — CHANGES_REQUESTED** [review]

File: aiomisc_dependency/__init__.py (hunk 1 above)
Comment (tzoiker) [reply aiomisc_dependency/__init__.py]: Not sure, `set` is not a sequence, for example

Comment (dizballanze) [reply aiomisc_dependency/__init__.py]: Ok, let's it be `Iterable`

Comment (tzoiker) [reply aiomisc_dependency/__init__.py]: Not a good practice, for example, generators are iterables

Comment (tzoiker) [reply aiomisc_dependency/__init__.py]: Forced to be a set or a sequence

**Review by dizballanze — APPROVED** [review]

**Comment by dizballanze** [comment]
[0.1.16](https://pypi.org/project/aiomisc-dependency/0.1.16/) released.
Thank you!

**Pull request merged**